core/domain/summary_services.py

```diff
@@ -372,9 +375,16 @@ def get_displayable_exp_summary_dicts(exploration_summaries):
         exploration_summary.id
         for exploration_summary in exploration_summaries]
 
-    view_counts = (
-        stats_jobs_continuous.StatisticsAggregator.get_views_multi(
-            exploration_ids))
+    if feconf.ENABLE_NEW_STATS_FRAMEWORK:
+        version_numbers = [
+            exploration_summary.version
+            for exploration_summary in exploration_summaries]
+        view_counts = stats_services.get_view_multi(
```

Comment:
Ah, I see why you changed the signature.

I suggest doing this a little differently. Call stats_services.get_exploration_stats_multi() or similar to get the exploration stats domain objects. Then do a list comprehension here to extract the start counts.

Reply:
@seanlip Why though? Are you suggesting not to have a get_views_multi method at all?

Reply:
Yes, that us what 'm suggesting, unless get_views_multi is needed elsewhere. Otherwise it bexomes a little awkward to pass stuff into it -- though if you have the ExpVersionReference domain object and use that, my concerns here are also alleviated.

Reply:
Done. Using get_exploration_stats_multi now.